openedx/features/caliper_tracking/caliper_config.py

```diff
@@ -1,10 +1,11 @@
 from openedx.features.caliper_tracking.transformers.bookmark_transformers import edx_bookmark_listed
-
```

Comment:
nit: remove extra empty line

Reply:
It has been already removed in the latest commit.